.github/workflows/test.yaml

```diff
@@ -177,10 +272,30 @@ jobs:
           # Stop local network services
           pkill hc-run-local
 
+      - name: Smoke test - trycp_write_validated with CHC enabled
+        run: |
+          set -x
+
+          # start the CHC reference implementation service
+          nix develop .#ci -c bash -c "hc-chc-service --port 8181 &"
```

Comment:
Could a section be added to the README on running the CHC service? There are a couple already that just give a recommended command for things like `hc-run-local-services` or the sandbox. It would be useful to have these two steps documented where you start the service and ask the sandbox to use it